README.md

```diff
@@ -109,6 +109,53 @@ Instead of `v-model` you can use the `index` prop and `change` event:
 />
 ```
 
+### Events
+
+`TinyBox` also emits the following events:
+ - `onPrev`/`onNext` with the next image being shown as a param
+ - `onClose` when `TinyBox` is closed
+
```

Comment:
Also, these are not the only events — tinybox also emits `change` every time the index changes — see it in action [here](https://github.com/NickKaramoff/vue-tinybox/blob/master/src/tinybox.vue#L263)